CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 * Add new `Rails/Blank` cop. ([@rrosenblum][])
 * Add new `Rails/Present` cop. ([@rrosenblum][])
 * [#4004](https://github.com/bbatsov/rubocop/issues/4004): Allow not treating comment lines as group separators in `Bundler/OrderedGems` cop. ([@konto-andrzeja][])
+* [#4145](https://github.com/bbatsov/rubocop/issues/4145): Add new `Style/YodaCondition` cop. ([@smakagon][])
```

Comment:
Due to yesterday's release you'll have to move this to the new `master` section of the changelog.